lib/Smokeping.pm

```diff
@@ -1960,7 +1961,11 @@ ALERT
             } else {
                         do_debuglog("Alert \"$_\": no match for target $name\n");
             }
-            $tree->{'prevmatch'.$s}{$_} = $match;
+            if ($match == 0) {
+                $tree->{'prevmatch'.$s}{$_} = $match;
+            } else {
+                $tree->{'prevmatch'.$s}{$_} += $match;
+            }
         }
```

Comment:
not worth it?
We find that info quite helpful while debugging network problems...